src/constants.js

```diff
@@ -0,0 +1 @@
+export const CHANNEL = '__glamorous__'
```

Comment:
Option 1 is certainly the simplest:

1. Disable eslint for this line.
2. Change `constants.js` to this:

```js
const CHANNEL = '__glamorous__';
export default CHANNEL;
```
And then change all the `{CHANNEL}` references to just `CHANNEL`.

3. Update the `package.json` to have this for the `eslintConfig` block:

```js
"eslintConfig": {
    "extends": [
      "kentcdodds",
      "kentcdodds/jest",
      "kentcdodds/react",
      "kentcdodds/prettier"
    ],
    "rules": {
      "import/prefer-default-export": 0
    }
  },
```

That third option will affect a rule we might want to keep around. @kentcdodds?